unicef/giga-data-ingestion
# feat: reconstruct database and update models

## What type of PR is this?

- `feat`: Commits that add a new feature

## Summary

What does this PR do

## How to test

1. Assert that you can `task migrate` cleanly
2. Assert that in the ingestion flow there should be a new `school_id_send_query_in` and `page_send_query_in` select field
3. Complete an add ingestion flow and assert that new entries were added to the database
https://github.com/unicef/giga-data-ingestion/assets/122899250/d487a7e5-8312-4970-a5e4-8a987b784cb6
